diff --git a/node_modules/@ionic/app-scripts/config/rollup.config.js b/node_modules/@ionic/app-scripts/config/rollup.config.js
index eed9dbf..25affa9 100644
--- a/node_modules/@ionic/app-scripts/config/rollup.config.js
+++ b/node_modules/@ionic/app-scripts/config/rollup.config.js
@@ -3,6 +3,7 @@ var commonjs = require('rollup-plugin-commonjs');
 var globals = require('rollup-plugin-node-globals');
 var builtins = require('rollup-plugin-node-builtins');
 var json = require('rollup-plugin-json');
+var babel = require('rollup-plugin-babel');


 // https://github.com/rollup/rollup/wiki/JavaScript-API
@@ -46,6 +47,7 @@ var rollupConfig = {
       extensions: ['.js']
     }),
     globals(),
+    babel({ exclude: 'node_modules/**'}),
     json()
   ]
